### DISCUSSION TITLE
polyfill.js and primefaces

### DISCUSSION THREAD

**Discussion by halliemorgan** [primefaces/community#2320] [PrimeFaces] [answered]
Has an analysis of the implications of the polyfill.js vulnerability?  I have examined the source of 12.0.0, there are 2 javascript libraries that indicate they are polyfill related (in the file title), but since the contents are minified its difficult to tell if the reference an external polyfill site.

**Comment by melloware** [comment, marked as answer]
polyfill was removed in 13.0.0 https://github.com/primefaces/primefaces/issues/10069  I suggest you upgrade to 13.0.10.
